scripts/fuzz_opt.py

```diff
@@ -556,7 +549,8 @@ def can_compare_to_others(self):
                 # NaNs can differ from wasm VMs
                 return not NANS
 
-        self.vms = [BinaryenInterpreter(), D8(), Wasm2C(), Wasm2C2Wasm()]
+        self.vms = [BinaryenInterpreter(), D8(), D8Liftoff(), D8TurboFan(),
```

Comment:
Is plain `D8()` still useful?

Reply:
I think so. That is how V8 normally runs, with (nondeterministic) tiering up, so it's good to test that, even if it is less valuable now.